packages/examples/bridge-reference/src/components/Header.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { useEffect } from "react";
-import { create, SdkConfig } from "@connext/sdk";
-import { ChainConfig } from "@connext/sdk/dist/config";
+import { create, SdkConfig } from "@connext/sdk-core";
+import { ChainConfig } from "@connext/sdk-core/dist/config";
```

Comment:
Why is this imported from the dist/? We should make sure this is a proper export if we need it imported in other places.